frontend/src/components/courses/evaluations/Stats.tsx

```diff
@@ -0,0 +1,148 @@
+import React from "react";
+import { PageTitle } from "../../utils/PageTitle";
+import { api, useBetterQuery } from "../../../services/apiService";
+import { Card, Container, Row, Col, ProgressBar } from "react-bootstrap";
+import { QuestionCategory, Question } from "../../../models/courses/question";
+import { StatsQuestion, Histogram } from "../../../models/courses/requests";
+import { PaginatedModalComment } from "./PaginatedModalComment";
+import { Loading } from "../../utils/Loading";
+
+const DigitToStar = (num: number) => {
+  let ceil = Math.ceil(num);
+  let floor = Math.floor(num);
+
+  let result: string = "★".repeat(floor);
+  let floating = ceil - num;
+
+  if (floating > 0.75) {
+    result += "★";
+  } else if (floating > 0.25) {
+    result += "⭑";
+  }
+
+  return result;
+};
+
+const DictToHistogram = (histogram: Histogram, sum: number) => (
+  <Col>
+    {Object.keys(histogram)
+      .sort()
+      .reverse()
+      .map((key) => (
+        <Row>
+          <Col sm={1}>
+            <p>{key}</p>
+          </Col>
+          <Col sm={10}>
+            <ProgressBar
+              now={(histogram[key] / sum) * 100}
+              key={"hist" + key}
+              label={key}
+              srOnly
+            />
+          </Col>
+        </Row>
+      ))}
+  </Col>
+);
+
+const StatsCardQuestion = ({ statsQuestion }) => {
+  let sum = 0;
+  Object.keys(statsQuestion.histogram)
+    .map((e) => parseInt(e))
+    .forEach((key) => {
+      sum += statsQuestion.histogram[key];
+    });
```

Comment:
Use `reduce` to calculate sums https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce